microsoft/PowerToys
# Do not load plugin when it is disabled

## Summary of the Pull Request

**What is this about:**
It does not make sense to load the plugin when it is disabled. For example, plugin loading always fails and show annoying popup. A user wants to disable a plugin in order not to see that popup again.

We do not load a plugin if it is disabled and we do not unload a plugin if it is disabled from the Plugin Manager.

**What is include in the PR:** 
- Do not load a plugin if it is disabled
- Move loading and initializing code to `PluginPair`
- Do not stop loading plugins if one of them failed
- Delete distinguishing between debug and release loading
- Localize dialog messages

**How does someone test / validate:** 
- Artificially fail a plugin. For example, throw in the constructor or Init method
- Verify that plugin failed to load at startup. A dialog message has to be shown.
- Go to settings and disable that plugin
- Restart PowerToys
- Verify that plugin is not loaded. A dialog message is not shown and it is not possible to invoke it from PT Run.

Test the Plugin Manager and PT Run for regressions

## Quality Checklist

- [X] **Linked issue:** #10405

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
